sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/EventGridKeyCredentialPolicy.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Messaging.EventGrid
+{
+    internal class EventGridKeyCredentialPolicy : AzureKeyCredentialPolicy
```

Comment:
Do you really need to inherit from the AzureKeyCredentialPolicy?

Reply:
I guess not